app/src/main/java/io/pslab/models/CompassData.java

```diff
@@ -0,0 +1,40 @@
+package io.pslab.models;
+
+public class CompassData {
+    private String Bx;
+    private String By;
+    private String Bz;
+
+    public CompassData() {
+    }
+
+    public CompassData(String x, String y, String z) {
+        this.Bx = x;
+        this.By = y;
+        this.Bz = z;
+    }
+
+    public String getBx() {
+        return Bx;
```

Comment:
Can you append `this.` to these?

Reply:
Are you suggesting to append this. to getters also or only in setters? @CloudyPadmal